megamek/src/megamek/common/QuirksHandler.java

```diff
@@ -446,6 +448,11 @@ public static void saveCustomQuirksList() throws IOException {
 
                 // Write out quirks
                 List<QuirkEntry> quirks = customQuirkMap.get(unitId);
+                if (quirks.isEmpty()) {
+                    output.write("\t\t" + getOpenTag(QUIRK));
+                    output.write(NO_QUIRKS);
+                    output.write(getCloseTag(QUIRK) + "\n");
+                }
                 for (QuirkEntry quirk : quirks) {
```

Comment:
I know this is a bit superfluous, but would this be best placed in an else statement for future readability?

Reply:
Sounds right, yes